### PR TITLE
Add Relay Kit

### PR DESCRIPTION
**Pull request by germartinez**
Add Relay Kit package

### CODE REVIEW

File: packages/relay-kit/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The Relay Kit allows to abstract users from the transaction fees payment (gas fees) allowing the use of native token or ERC-20 tokens. This will enable you to pay transaction fees directly from funding available in a Safe.
+
+## Reference
+
+- [Relay Kit docs](https://docs.safe.global/learn/safe-core-account-abstraction-sdk/relay-kit)
+
+## License
+
+This library is [released under MIT](https://github.com/safe-global/account-abstraction-sdk/blob/main/LICENSE.md).
```
Comment (dasanra) [inline packages/relay-kit/README.md:15]:
```suggestion
This library is [released under MIT](https://github.com/safe-global/safe-core-sdk/blob/main/LICENSE.md).
```

File: packages/relay-kit/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Relay Kit
+
+[![NPM Version](https://badge.fury.io/js/%40safe-global%2Frelay-kit.svg)](https://badge.fury.io/js/%40safe-global%2Frelay-kit)
+[![GitHub Release](https://img.shields.io/github/release/safe-global/account-abstraction-sdk.svg?style=flat)](https://github.com/safe-global/account-abstraction-sdk/releases)
+[![GitHub](https://img.shields.io/github/license/safe-global/account-abstraction-sdk)](https://github.com/safe-global/account-abstraction-sdk/blob/main/LICENSE.md)
```
Comment (dasanra) [inline packages/relay-kit/README.md:5]:
```suggestion
[![GitHub](https://img.shields.io/github/license/safe-global/account-abstraction-sdk)](https://github.com/safe-global/safe-core-sdk/blob/main/LICENSE.md)
```

File: packages/relay-kit/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Relay Kit
+
+[![NPM Version](https://badge.fury.io/js/%40safe-global%2Frelay-kit.svg)](https://badge.fury.io/js/%40safe-global%2Frelay-kit)
+[![GitHub Release](https://img.shields.io/github/release/safe-global/account-abstraction-sdk.svg?style=flat)](https://github.com/safe-global/account-abstraction-sdk/releases)
```
Comment (dasanra) [inline packages/relay-kit/README.md:4]:
```suggestion
[![GitHub Release](https://img.shields.io/github/release/safe-global/account-abstraction-sdk.svg?style=flat)](https://github.com/safe-global/safe-core-sdk/releases)
```

**Comment by germartinez** [comment]
True, I forgot about the README file. Thanks @dasanra!

**Review by dasanra — APPROVED** [review]

**Pull request merged**